go/control/api/api.go

```diff
@@ -108,12 +112,6 @@ type IdentityStatus struct {
 	// Node is the node identity public key.
 	Node signature.PublicKey `json:"node"`
 
-	// P2P is the public key used for p2p communication.
-	P2P signature.PublicKey `json:"p2p"`
-
-	// Consensus is the consensus public key.
-	Consensus signature.PublicKey `json:"consensus"`
```

Comment:
Where is the consensus key reported now? This is unrelated to P2P.

Reply:
Under `consensus.p2p`, so that it is concise with libp2p.

```
➜  compute-1 oasis-node control status
{
  "p2p": {
    "pub_key": "mrhfjjWQbz6VDNKWCHnLhwWQ5pzUX8Art3AcGhIl+V4=",
  }
},
{
  "consensus": {
    "p2p": {
      "pub_key": "dE9f9JFWYgfSYnZJmfF362516lZsl5OTWhMyk+b7LI8=",
    }
  }
}
```

Reply:
But this (e.g. what was in `Consensus`) is not the consensus P2P key. So there are two distinct keys:
* The P2P key (used for both libp2p and Tendermint P2P).
* The consensus key (used for signing consensus votes).

So you probably want to add the consensus key back.

Reply:
I guess I misunderstood that. Will change back.